core/src/main/resources/hudson/ProxyConfiguration/help-name_ja.html

```diff
@@ -1,14 +1,20 @@
 <div>
   Jenkinsサーバーがファイアーウォールの内側にあってインターネットに直接アクセスできない上に、
-  サーバーのJVMがインターネットと接続できるように適切(<a  href="https://docs.oracle.com/javase/8/docs/technotes/guides/net/properties.html">詳細はネットワークのプロパティを参照</a>)に設定されていない場合、
+  サーバーのJVMがインターネットと接続できるように適切(<a
+    href="https://docs.oracle.com/javase/8/docs/technotes/guides/net/properties.html"
+    >詳細はネットワークのプロパティを参照</a
+  >)に設定されていない場合、
```

Comment:
What is the justification for formatting this poor? We're also reflowing text, so it cannot be to have attributes on separate lines from `>` to make diffs clearer.

Reply:
https://github.com/prettier/prettier/issues/8302 indirectly answers this: In long lines, inside tags just before `>` can be the only location to break long lines safely.

Also https://prettier.io/blog/2018/11/07/1.15.0.html#whitespace-sensitive-formatting

Reply:
Looks better now, thanks for the link. I checked and our help files are all block layout.